opencti-platform/opencti-front/src/private/components/settings/users/UserCreation.jsx

```diff
@@ -192,10 +184,12 @@ class UserCreation extends Component {
                 password: '',
                 confirmation: '',
                 objectOrganization: [],
+                account_status: 'Active',
```

Comment:
translation?

Reply:
@Archidoit - Stored internally as 'Active' regardless of selected language selected. On display, the MenuItem display value will translate this with the correction you identified.

Reply:
yes you're right!